tests/ui/specs/VideoPlayer/VideoPlayer-specs.js

```diff
@@ -0,0 +1,282 @@
+		});
+
+		it('should not display controls', function () {
+			Page.delay(1000);
+			expect(videoPlayerDisabled.mediaControlsFrame.getCSSProperty('opacity').value).to.equal(0);
+		});
+
+		it('should not display controls on click', function () {
+			Page.delay(1000);
+			videoPlayerDisabled.videoPlayerContainer.click();
+			Page.delay(250);
+			expect(videoPlayerDisabled.mediaControlsFrame.getCSSProperty('opacity').value).to.equal(0);
+		});
+
+		it('should display poster', function () {
+			Page.delay(1000);
+			expect(videoPlayerDisabled.video.getAttribute('poster')).to.equal('http://media.w3.org/2010/05/sintel/poster.png');
+		});
+
+		describe('5-way', function () {
+			it('should not display controls on 5-way down', function () {
+				Page.delay(1000);
+				Page.spotlightDown();
+				Page.delay(250);
+				expect(videoPlayerDisabled.mediaControlsFrame.getCSSProperty('opacity').value).to.equal(0);
+			});
+		});
+	});
+
+
+	describe('spotlightDisabled', function () {
+		beforeEach(function () {
+			Page.open('SpotlightDisabled');
+		});
+
+		it('should not have the play/pause button focused', function () {
+			Page.delay(1000);
+
+			expect(videoPlayerSpotlightDisabled.playButton.isFocused()).to.not.be.true();
+		});
+
+		it('should play media on playButton click', function () {
+			Page.delay(1000);
+			expect(videoPlayerSpotlightDisabled.slider.getCSSProperty('width').value).to.equal('0px');
+
+			videoPlayerSpotlightDisabled.playButton.click();
+			Page.delay(2000);
+
+			expect(videoPlayerSpotlightDisabled.slider.getCSSProperty('width').value).to.not.equal('0px');
+		});
+
+		it('should skip forwards/backwards on previous/next button click', function () {
+			Page.delay(1000);
+			expect(videoPlayerSpotlightDisabled.slider.getCSSProperty('width').value).to.equal('0px');
+			videoPlayerSpotlightDisabled.nextButton.click();
+
+			expect(videoPlayerSpotlightDisabled.slider.getCSSProperty('width').value).to.not.equal('0px');
+			videoPlayerSpotlightDisabled.previousButton.click();
+			expect(videoPlayerSpotlightDisabled.slider.getCSSProperty('width').value).to.equal('0px');
+		});
+
+		describe('5-way', function () {
+			it('should not show controls on 5-way down', function () {
+				Page.delay(1000);
+
+				if (videoPlayerSpotlightDisabled.mediaControlsFrame.getCSSProperty('opacity').value === 1) {
+					videoPlayerSpotlightDisabled.videoPlayerContainer.click();
+					Page.delay(250);
+				}
+
+				expect(videoPlayerSpotlightDisabled.mediaControlsFrame.getCSSProperty('opacity').value).to.equal(0);
+				Page.spotlightDown();
+				Page.delay(250);
+				expect(videoPlayerSpotlightDisabled.mediaControlsFrame.getCSSProperty('opacity').value).to.equal(0);
+			});
+		});
+	});
+
+	describe('props', function () {
+		beforeEach(function () {
+			Page.open('Props');
+		});
+
+		it('should hide controls after 3 seconds', function () {
+			Page.delay(3500);
+			expect(videoPlayerProps.mediaControlsFrame.getCSSProperty('opacity').value).to.equal(0);
```

Comment:
here maybe you can show that the controls are first displayed, then they dissapear